QUARKUS-2387.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-2387 - Promote Mailer extension from TP to full support
+
+Evaluate the extension for promotion to full support. Propose needed enhancements and report found bugs. 
+
+## Scope of the testing
+ - Review of test coverage in Quarkus integration tests
+ - Review upstream guides for the extension
+ - Exploratory testing of the extension
```

Comment:
I should probably stress out that I will use gmail config with the exploratory testing

Reply:
done